src/skin/popup.html

```diff
@@ -122,17 +122,30 @@ <h2 id="title-name"><span class="i18n_name"></span></h2>
 </div>
 
 <div id="blockedResourcesContainer">
-  <p id="pbInstructions">
+  <p class="pbInstructions">
     <span id="instructions-many-trackers"></span>
     <span id="instructions-no-trackers" style="display:none">
       <span class="i18n_popup_instructions_no_trackers" data-i18n_contents_placeholders="<a target='_blank' title='i18n_what_is_a_tracker' class='tooltip' href='https://privacybadger.org/#What-is-a-third-party-tracker'>"></span>
       <span id="no-third-parties" class="i18n_popup_blocked" style="display:none"></span>
     </span>
+    <div id="toggle-blocked-resources-container" tabIndex="0">
+      <span id="expand-blocked-resources" title="i18n_show_more_information" class="ui-icon ui-icon-caret-d tooltip"></span>
+      <span id="collapse-blocked-resources" title="i18n_show_more_information" class="ui-icon ui-icon-caret-u tooltip"></span>
+    </div>
```

Comment:
Collapsing shouldn't get the same tooltip.

Maybe instead of tooltips, we should add a hover effect to make it more clear you can click on the header to get more information. Maybe the caret should move a little.

Reply:
Yeah, let's do a hover effect over the clickable area. More clear/helpful than tooltips, no translation required.